model-optimizer/mo/utils/ir_reader/extenders/conv_extender.py

```diff
@@ -50,3 +50,7 @@ def extend(op: Node):
         op['pad'] = int64_array(pad)
 
         op['spatial_dims'] = [i + 2 for i in range(dim)]
+
+        # We need this workaround to not to break shape inference function
+        if op.has_valid('auto_pad') and op.auto_pad == 'explicit':
+            op['auto_pad'] = None
```

Comment:
Instead of doing this we should change the shape inference function of Pooling. The same comment for the file below.

Reply:
Ok, now this case with `explicit` value for `auto_pad` attribute resolved in shape infer functions.